INSTALL-vagrant.md

```diff
@@ -22,29 +22,36 @@ cp configs/vagrant-example.yaml configs/vagrant.yaml
  - Domain (`hitchwiki.test` by default) or develop using IP (`192.168.33.10` by default)
 
 #### Install
-1. Start installation script
+1. Run the installation script
 ```bash
 ./scripts/vagrant/install.sh
 ```
 
+This will
+* Install [Vagrant hostmanager](https://github.com/smdahlen/vagrant-hostmanager) plugin
+* Setup Vagrant box (optional)
+
 2. Install will ask for your password to add `hitchwiki.test` to your `/etc/hosts` file.
 You can [modify your sudoers file](https://github.com/smdahlen/vagrant-hostmanager#passwordless-sudo) to stop Vagrant asking for password each time.
 
 3. Open [http://hitchwiki.test/](http://hitchwiki.test/) in your browser. [*https*://hitchwiki.test/](https://hitchwiki.test/) works if you set `setup_ssl` to `true` from `configs/vagrant.yaml`
 
 After setup your virtual machine is running. Suspend the virtual machine by typing `vagrant suspend`.
-When you're ready to begin working again, just run `vagrant up`.
-
-
-#### Install script will do the following:
-* Install [Vagrant hostmanager](https://github.com/smdahlen/vagrant-hostmanager) plugin
-* Setup Vagrant box (optional)
+When you're ready to begin working again, just run `vagrant provision`.
+
+#### Ansible
+As soon as ssh is ready, [Ansible](https://docs.ansible.com/ansible/latest/intro.html) takes over to configure your system according to the `hitchwiki.yml` [Playbook](https://docs.ansible.com/ansible/latest/playbooks_intro.html):
```

Comment:
_"As soon as ssh is ready"_ => _"As soon as Vagrant machine is ready"_ probably more clear for anyone new to Vagrant.

Reply:
done